src/Context/navbar-context.js

```diff
@@ -0,0 +1,25 @@
+import { createContext, useContext, useState } from "react";
+
+const NavbarContext = createContext();
+
+const NavbarProvider = ({ children }) => {
+  const encodedToken = localStorage.getItem("ENCODED_TOKEN_2");
+  const [navbarButtonText, setNavbarButtonText] = useState(
```

Comment:
there is no need in creating a navbar context, the context or even localstorage can be used to store the userToken